src/NzbDrone.Core/NzbDrone.Core.csproj

```diff
@@ -970,6 +970,10 @@
       <Link>MediaInfo.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\MediaInfoDotNet.dll.config">
+      <Link>MediaInfoDotNet.dll.config</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
@markus101 Yes, but this part has to be removed, was changed to NzbDrone.Core.dll.config in the mediainfo fix.